src/page/Home.tsx

```diff
@@ -15,20 +17,132 @@ class Home extends React.Component<{}>
 
     public render() {
         return (
-            <div className="text-center">
-                <Row>
-                    <Col lg="8" className="m-auto">
-                        <SearchBox
-                            onChange={this.onSearch}
-                            placeholder="Search variants"
-                        />
-                    </Col>
-                </Row>
-                <Row className="py-4">
-                    <Col className="m-auto">
-                        {this.inputText}
-                    </Col>
-                </Row>
+            <div>
+                <div className="text-center">
+                    <Row>
+                        <Col lg="8" className="home-logo">
+                            <GenomeNexusLogo imageHeight={130}/> 
+                                GenomeNexus
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="8" className="home-description">
+                            A resource for annotation and interpretation of genetic variants
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="6" className="m-auto">
+                            <SearchBox
+                                onChange={this.onSearch}
+                                placeholder="Search variants"
+                                searchIconClassName="fa fa-search"
+                            />
+                        </Col>
+                    </Row>
+                    <Row>
+                        <Col lg="7" className="home-function-description">
+                            Genome Nexus will integrate information from a variety of existing resources, 
+                            including databases that convert DNA changes to protein changes, predict the 
+                            functional effects of protein mutations, and contain information about mutation 
+                            frequencies, gene function, variant effects, and clinical actionability.
+                        </Col>
+                    </Row>
+                    <Row className="justify-content-md-center">
+                        <Col lg="2">
+                            <Button onClick={() => window.open("http://genomenexus.org/swagger-ui.html", "_blank")} variant="outline-primary">Try live API</Button>
```

Comment:
Instead of registering a callback which refers to a global `window` object, can we do something simpler like this:

```suggestion
                            <Button href={"http://genomenexus.org/swagger-ui.html"} variant="outline-primary">Try live API</Button>
```